src/Client.ts

```diff
@@ -20,9 +19,11 @@ import { DDiscord, DOption, DSlash } from "./decorators";
 import { GuildNotFoundError } from "./errors";
 
 export class Client extends ClientJS {
+  botid?: string;
+  static importedClass: string[] = []
```

Comment:
Make private and add getter+setter. Also, should be plural.
```suggestion
  private static _importedClasses: string[] = []
```

Reply:
this is removed cause this method did not work on testings, new method introduce for now